cmd/juju/cloud/add.go

```diff
@@ -182,7 +186,7 @@ func queryName(cloudMetadataStore CloudMetadataStore, pollster *interact.Pollste
 	}
 
 	for {
-		name, err := pollster.Enter("a name for the cloud")
+		name, err := pollster.Enter(fmt.Sprintf("a name for your %s", cloudType))
```

Comment:
Is this clearer as "a name for your %s cloud". It's a little funny for 'maas', but makes more sense for 'manual' and is fine with 'openstack', 'vsphere', and so on.